<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-CQO0GVPR" failures="2" tests="2" name="BankingApp Test(failed)" time="21.041" errors="0" timestamp="2023-06-03T00:26:47 AEST">
  <testcase classname="com.inetbanking.testCases.TC_LoginTestDDT_002" name="logintest" time="8.913">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetbanking.testCases.TC_LoginTestDDT_002.logintest(TC_LoginTestDDT_002.java:51)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- logintest -->
  <testcase classname="com.inetbanking.testCases.TC_LoginTestDDT_002" name="logintest" time="6.077">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetbanking.testCases.TC_LoginTestDDT_002.logintest(TC_LoginTestDDT_002.java:51)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- logintest -->
</testsuite> <!-- BankingApp Test(failed) -->
